app/api/dao/admin.py

```diff
@@ -19,12 +19,21 @@ def assign_new_user(user_id, data):
         Returns:
             message: A message corresponding to the completed action.
         """
-
         new_admin_user_id = data['user_id']
 
         if user_id is new_admin_user_id:
             return messages.USER_CANNOT_BE_ASSIGNED_ADMIN_BY_USER, 403
 
+        #For test purposes
```

Comment:
You can remove this comment, because this is not only for tests, but actually "production" code that was missing ;)